python/tk_multi_reviewsubmission/submitter.py

```diff
@@ -22,7 +22,7 @@ def __init__(self):
         """
         self.__app = sgtk.platform.current_bundle()
     
-    def submit_version(self, path_to_frames, path_to_movie, thumbnail_path, sg_publishes,
+    def submit_version(self, version_name, path_to_frames, path_to_movie, thumbnail_path, sg_publishes,
```

Comment:
I'd prefer to see the new argument added to the end of the list rather than at the head. That will maintain argument ordering. Also, in this case I'd probably default version_name to None after moving it to the end of the list. You have fallback logic included below that can allow us to maintain the same default behavior as before, but allowing the version_name to be passed in if a template was defined.